tests/test_opds2_import.py

```diff
@@ -198,7 +198,7 @@ def test(self):
             moby_dick_delivery_mechanism.delivery_mechanism.drm_scheme)
         assert (
             MediaTypes.EPUB_MEDIA_TYPE ==
-            moby_dick_delivery_mechanism.delivery_mechanism.media_type)
+            moby_dick_delivery_mechanism.delivery_mechanism.content_type)
```

Comment:
I think this is also just a mistake in the tests. This is referencing the loop variable [`media_type` set in licensing.py](https://github.com/NYPL-Simplified/server_core/blob/develop/model/licensing.py#L1495).